Week3/homework/step3.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+  function addition(x) {
+    return base + x;
+  }
+  return addition;
```

Comment:
Fine. But you can also return an anonymous function like this:

```js
function createBase(base) {
  return function(x) {
    return base + x;
  };
}
```

Or, if you are adventurous:

```js
const createBase = base => x => base + x;
```

Reply:
I am not so adventurous for now. but the anonymous function nice. I changed my code.